docs/tutorial/part-eight/index.md

```diff
@@ -198,9 +198,9 @@ class Application extends React.Component {
 
 ## Keep making it better
 
-In this section we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
+In this section, we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
 
-Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keep making your site better!
+Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keeps making your site better!
```

Comment:
_keep_ seems correct in this instance?

Reply:
it relates to "it" so it is in the third person.

Reply:
The latter half of the sentence (after the and): "keep making your site better" relates to the reader and asks them to _keep_ making their site better by continually looking through the feedback!

Reply:
Yeah, that's right, I haven't got it earlier, but now I do. Gonna do a little fix!

Reply:
No worries. Better grammar through collaboration! 🙌